src/components/IconGithub.jsx

```diff
@@ -1,16 +1,41 @@
+import React, { useState, useEffect } from 'react';
+
 export default function IconGithub() {
+  const [stars, setStars] = useState(0);
+
+  useEffect(() => {
+    const cachedStarCount = localStorage.getItem('starCount');
```

Comment:
I think session storage would be better so the user gets the update value for each session.